utils/usage.py

```diff
@@ -1,16 +1,6 @@
 import argparse
```

Comment:
I was thinking we should rename this file. maybe `report_block_usage`

Reply:
Sure